tests/teamscale_client_test.py

```diff
@@ -16,163 +12,238 @@
     FileSystemSourceCodeConnectorConfiguration, ProjectConfiguration, FindingDescription
 from teamscale_client.utils import to_json
 
+TEAMSCALE_TEST_VERSION = "v8.0.0"
+PROJECT = "foo"
+USER = "admin"
+ACCESS_TOKEN = USER
+SUCCESS_TEXT = "success"
```

Comment:
AFAICS none of the new services return "success" anymore. They all (at least should) purely communicate the success or failure of an operation via the HTTP response code.